box_it/index.js

```diff
@@ -0,0 +1,87 @@
+//Test data for boxit
```

Comment:
Should be using 
```node 
process.argv
```
 to retrieve inputs from command line and print those inputs within the boxes